vector_initializer.py

```diff
@@ -21,63 +18,45 @@ def initialize_vector(self, length, pop_size):  # Should it be normalized?
 
 
 class XavierInitializer(VectorInitializer):
-    """XavierInitializer
-
-        Initialize each element of the vector with value from U [-(1/sqrt(n)), 1/sqrt(n)] distribution
-    """
-
-    def __init__(self, seed):
-        super().__init__(seed)
+    """Initialize each element of the vector with value from U [-(1/sqrt(n)), 1/sqrt(n)] distribution."""
 
     def initialize_vector(self, length, pop_size):
         return random.uniform(-1.0/sqrt(pop_size), 1.0/sqrt(pop_size), length)
 
 
 class HeInitializer(VectorInitializer):
-    """HeInitializer
-
-        Initialize each element of the vector with value from N (0.0, sqrt(2/n)) distribution
-    """
-
-    def __init__(self, seed):
-        super().__init__(seed)
+    """Initialize each element of the vector with value from N (0.0, sqrt(2/n)) distribution."""
 
     def initialize_vector(self, length, pop_size):
         return random.normal(0, sqrt(2.0/pop_size), length)
 
 
 class RandomInitializer(VectorInitializer):
-    """RandomInitializer
-
-        Initialize each element of the vector with random value
-    """
-    def __init__(self, seed):
-        super().__init__(seed)
+    """Initialize each element of the vector with random value."""
 
     def initialize_vector(self, length, pop_size):
         return random.rand(length)
 
 
 class ConstInitializer(VectorInitializer):
-    """ConstInitializer
-
-        Initialize each element of the vector with constant value specified as parameter of the constructor
-    """
-    def __init__(self, value):
-        super().__init__()
+    """Initialize each element of the vector with constant value specified as parameter of the constructor."""
+    def __init__(self, seed, value):
+        super().__init__(seed)
         self.value = value
 
     def initialize_vector(self, length, pop_size):
         return np.full(length, self.value)
 
 
 class ZeroInitializer(VectorInitializer):
-    """RandomInitializer
-
-        Initialize each element of the vector with zero
-    """
-    def __init__(self):
-        super().__init__()
+    """Initialize each element of the vector with zero"""
 
     def initialize_vector(self, length, pop_size):
         return np.zeros(length)
+
+
+class ReverseSquareInitializer(VectorInitializer):
```

Comment:
Shouldn't it be `ReverseSquareRoorInitializer`? I was thinking about using `ConstInitializer` with value specified in the constructor for this purpose but we can have separate initializer for it as well :)

Reply:
yes, you right :)